src/main/resources/jupyter/cluster-site.conf

```diff
@@ -12,18 +12,33 @@
     RewriteEngine on
 
     # RStudio
+
     RewriteCond %{HTTP:Upgrade} =websocket
-    RewriteCond %{REQUEST_URI} /notebooks/[^/]*/[^/]*/rstudio/.* [NC]
-    RewriteRule /notebooks/[^/]*/[^/]*/rstudio/(.*)     ws://127.0.0.1:8001/$1  [P,L]
+    RewriteCond %{REQUEST_URI} /proxy/[^/]*/[^/]*/rstudio/.* [NC]
+    RewriteRule /proxy/[^/]*/[^/]*/rstudio/(.*)     ws://127.0.0.1:8001/$1  [P,L]
 
     RewriteCond %{HTTP:Upgrade} !=websocket
-    RewriteCond %{REQUEST_URI} /notebooks/[^/]*/[^/]*/rstudio/.* [NC]
-    RewriteRule /notebooks/[^/]*/[^/]*/rstudio/(.*)     http://127.0.0.1:8001/$1 [P,L]
+    RewriteCond %{REQUEST_URI} /proxy/[^/]*/[^/]*/rstudio/.* [NC]
```

Comment:
as discussed, would like a way to not require the trailing slash. i've tried things like 

```
RewriteCond %{REQUEST_URI} /proxy/[^/]*/[^/]*/rstudio/.* [NC,OR]
RewriteCond %{REQUEST_URI} /proxy/[^/]*/[^/]*/rstudio [NC]
```
and other variations but no luck. the hammer approach may be to just duplicate these blocks entirely

Reply:
There must be a solution here, I'm just not sure offhand. Can play around a little..

Reply:
Ok, I don't think that needs to block this PR, we can have a follow-on task